resources/user-agents/browsers/firefox/firefox-3-8.json

```diff
@@ -1,109 +0,0 @@
-{
```

Comment:
As far as I can tell, there was never a 3.8 release tagged (as alpha or otherwise). In our logs I only found 2 useragents with a Firefox 3.8 version number (this is since 2009 or so).

Firefox's version was bumped to 4.0 during the 3.7 alpha release cycle, so I don't see why 3.8 would have ever been tagged.